XGenerate/src/test/resources/features/unit/config/model/Unit_Config_Model_ModelAttributeInjection.feature

```diff
@@ -267,5 +265,5 @@ Feature: Unit_Config_Model_ModelAttributeInjection
     When I run the generator
     Then I expect the following error message:
       """
-      com.xbreeze.xgenerate.model.ModelPreprocessorException: Error while processing model attribute injection for target XPath ´concats('The entity was ', @name)´: Syntax error after or around the end of ´concats´
+      com.xbreeze.xgenerate.model.ModelPreprocessorException: Error while processing model attribute injection for target XPath ´concats('The entity was ', @name)´: net.sf.saxon.trans.XPathException: Cannot find a 2-argument function named Q{http://www.w3.org/2005/xpath-functions}concats()
```

Comment:
Can you only include the message, without the class of saxon in it?

Reply:
I don't think this is easily done since this is the result of the getMessage invoked on the exception.